HW_1/Task_1.py

```diff
@@ -0,0 +1,21 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+username = input('Введите ваше имя: ')
+
+x = int(input('Введите трехзначное число: '))
+a = int(x//100)
+b = int(x/10%10)
```

Comment:
Тут тоже можно обойтись без этой функции.
```suggestion
b = x // 10 % 10
```